src/React/Basic/DOM.purs

```diff
@@ -682,6 +683,10 @@ type Props_form =
   , children :: Array JSX
   , method :: String
   , name :: String
+  , onChange :: EventHandler
+  , onInput :: String
+  , onInvalid :: String
+  , onSubmit :: String
```

Comment:
Shouldn't this be an `EventHandler`? (and I think also the other ones?)

Reply:
time to PR upstream again probably :100:

Reply:
Nah, in the upstream there was no place to define these things AFAIK